share/spice/just_delete_me/content.handlebars

```diff
@@ -0,0 +1,15 @@
+{{#if notes}}
+  <p>
+    {{notes}}
+  </p>
+{{/if}}
+{{#if delete_url}}
+  <p>
+    <b>URL:</b> <a href='{{delete_url}}'>{{delete_url}}</a>
+  </p>
+{{/if}}
+{{#if email}}
+  <p>
+    <b>Email:</b> <a href='mailto:{{email}}'>{{email}}</a>
```

Comment:
Remove the `<b>`

Is the user supposed to send an email here requesting their account to be deleted? It's unclear what this email address is for...